lesson04/assignment/css/style.scss

```diff
@@ -0,0 +1,141 @@
+// Colors and basic styling
+
+$dkolive: #3e442b;
+$nickel: #6a7062;
+$silver: #928a9c;
+$lavender: #c7bed9;
+$ltblue: #dcf2ff;
+
+// Mixin for making box shadows
+
+@mixin shadow {
+  box-shadow: .3rem .3rem .2rem #2a2f1d;
+}
+
+body {
+  background-color: $dkolive;
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 1rem;
+  box-sizing: border-box;
+  margin: 0;
+  border: 0;
+  padding: 3rem;
+}
+
+p {
+  display: block;
+  padding: .2rem;
+}
+
+a {
+  display: block;
+  padding: .2rem;
+  background-color: $ltblue;
+}
+
+// Menu container
+
+.menu {
+  display: block;
+  margin: 0;
+  border: 0;
+  padding: .5rem;
+  width: 12rem;
+  height: 28rem;
+
+  // Menu items
+
+  &__cover {
+    margin: .5rem;
+    border: .5rem ridge $nickel;
+    border-radius: .25rem;
+    padding: .5rem;
+    text-align: center;
+    background-color: $silver;
+    color: $ltblue;
+    font-size: 1.5rem;
+    font-weight: bold;
+    z-index: 6;
+    position: relative;
+    will-change: transform;
+    @include shadow;
+  }
+
+  &__opt1 {
+    @extend .menu__cover;
+    font-size: 1.2rem;
+    z-index: 5;
+    top: -4rem;
+  }
+  &__opt2 {
+    @extend .menu__cover;
+    font-size: 1.2rem;
+    z-index: 4;
+    top: -7.9rem;
+  }
+  &__opt3 {
+    @extend .menu__cover;
+    font-size: 1.2rem;
+    z-index: 3;
+    top: -11.8rem;
+  }
+  &__opt4 {
+    @extend .menu__cover;
+    font-size: 1.2rem;
+    z-index: 2;
+    top: -15.7rem;
+  }
+  &__opt5 {
+    @extend .menu__cover;
+    font-size: 1.2rem;
+    z-index: 1;
+    top: -19.6rem;
+  }
+
+  // Transition effects on hover
+
+  &:hover > div {
+    transition: transform 1s ease-in;
+
```

Comment:
You have your ease-in, but will need an ease-out. Will probably be easiest to fade out instead of reverse the animation you have. Check out this article: https://stackoverflow.com/questions/16878977/how-to-fade-out-after-hover-is-done-using-css